cibuildwheel/logger.py

```diff
@@ -262,8 +257,6 @@ def file_supports_unicode(file_obj: IO[AnyStr]) -> bool:
     return "utf" in codec_info.name
 
 
-"""
-Global instance of the Logger.
-"""
+# Global instance of the Logger.
 # (there's only one stdout per-process, so a global instance is justified)
 log = Logger()
```

Comment:
Instances don't have have docstrings, only modules, classes, and functions (and in slots in Python 3.8+). I think there might be a bit of syntax somewhere kind of like this, like for class instances or something, because I've written things like that before, but wasn't able to get it just now with a little trying.